### ISSUE TITLE
Less findings than expected

### ISSUE THREAD

**Issue by ruedbi** [dougpuob/cppnamelint#61]
Note: This is a follow up for the  0.3.4 fix.

I did some checks with the new version and there are far less findings than I would expect.
Again I used  https://github.com/changkun/modern-cpp-tutorial for you to reproduce.
I also used the default toml config file from the cppnamelint repo.
I reduced the scope of the tests arbitrarily to §10 in the tutorial.
What I get:
```
cppnamelint utility v0.3.4
---------------------------------------------------
 File    = 10.1.without.concepts.cpp
 Config  = cppnamelint.toml
 Checked =     2  [File:0 | Func:  1 | Param:  0 | Var:  1 | Enum:  0 | Struct:  0 | Union:  0 | Class:  0]
 Error   =     1  [File:0 | Func:  0 | Param:  0 | Var:  1 | Enum:  0 | Struct:  0 | Union:  0 | Class:  0]
------------------------------------------------------------
  <  15,    5> Variable  : l (std::list<int>)

cppnamelint utility v0.3.4
---------------------------------------------------
 File    = 10.2.concepts.cpp
 Config  = cppnamelint.toml
 Checked =     0  [File:0 | Func:  0 | Param:  0 | Var:  0 | Enum:  0 | Struct:  0 | Union:  0 | Class:  0]
 Error   =     0  [File:0 | Func:  0 | Param:  0 | Var:  0 | Enum:  0 | Struct:  0 | Union:  0 | Class:  0]
------------------------------------------------------------

```
The first file report looks reasonable, but in the second there are **no checks at all.** Not evene the "struct Person" is detected.

I cross checked with YouPlayGround.cpp from the repo:
```
cppnamelint utility v0.3.4
---------------------------------------------------
 File    = YouPlayGround.cpp
 Config  = cppnamelint.toml
 Checked =     6  [File:0 | Func:  5 | Param:  1 | Var:  0 | Enum:  0 | Struct:  0 | Union:  0 | Class:  0]
 Error   =     1  [File:0 | Func:  0 | Param:  1 | Var:  0 | Enum:  0 | Struct:  0 | Union:  0 | Class:  0]
------------------------------------------------------------
  <  16,   25> Parameter : InputC (int)

```
this looks as expected

Please also note that the "-logfile" option causes cppnamelint to coredump / die with signal 11.

Can you please check? Thanks!

**Comment by dougpuob** [comment]
Hi @ruedbi 

- I tried the latest master (build with llvmorg-11.1.0), but different with your result. So I added a script to check all source files of `https://github.com/changkun/modern-cpp-tutorial` project. The following link is the log from Azure CI/CD for your reference. (
https://dev.azure.com/HelloClangTool/cppnamelint/_build/results?buildId=99&view=logs&j=5a2a871c-9779-595f-8e34-7d7186f2c3b5&t=b9dd42ce-5904-54ff-0f15-4330cfc452c1)

- Another issue you mentioned about the logger, has been fixed with #59 .

**Comment by dougpuob** [comment]
The two attachments, one is log file, another is output of the program on Ubuntu.
[modern-cpp-tutorial.txt](https://github.com/dougpuob/cppnamelint/files/6329073/modern-cpp-tutorial.txt)
[modern-cpp-tutorial.log](https://github.com/dougpuob/cppnamelint/files/6329074/modern-cpp-tutorial.log)

**Comment by ruedbi** [comment]
Hi Douglas, that looks much more reasonable. I will try to reproduce that. Thanks!

**Comment by ruedbi** [comment]
I can reproduce the results. Thanks!
Will make another ticket for new findings ;-)
